tests/integrations/huggingface/cassettes/test_huggingface_inference_client/test_huggingface_fill_mask.yaml

```
@@ -1,4 +1,357 @@
 interactions:
+    body: ''
+    headers:
+      accept:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)